core/src/common/network_abstraction_contiki.c

```diff
@@ -430,10 +430,10 @@ NetworkSocketError NetworkSocket_GetError(NetworkSocket * networkSocket)
 int NetworkSocket_GetFileDescriptor(NetworkSocket * networkSocket)
 {
     int result = -1;
-    if (networkSocket)
-    {
-        result = (int)networkSocket->Socket;
-    }
+//    if (networkSocket)
```

Comment:
This is for contiki there are no file descriptor and originally I was trying to be compact with similar logic. But I was getting warnings when compiling so decided to comment it out

This function exists so linux daemons have access to file descriptor so it can be passed into poll